src/drivers/bitbucket_cloud.js

```diff
@@ -150,14 +177,14 @@ class BitBucketCloud {
 
   async request(opts = {}) {
     const { token, api } = this;
-    const { endpoint, method = 'GET', body } = opts;
-
-    if (!endpoint) throw new Error('BitBucket Cloud API endpoint not found');
+    const { fullEndpoint, endpoint, method = 'GET', body } = opts;
+    if (!(endpoint || fullEndpoint))
```

Comment:
would not be better uri or url vs fullEndpoint?

Reply:
Yes